UIKitVK/UIKitVK/Extension/UIImageView+Extension.swift

```diff
@@ -0,0 +1,18 @@
+// UIImageView+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Расширение для получения аватара
+extension UIImageView {
+    func loadAvatar(url: String) {
```

Comment:
```suggestion
    func loadImage(url: String) {
```
Только Аватар загружаем??